libcontainer/nsenter/cloned_binary.c

```diff
@@ -372,35 +372,35 @@ static int seal_execfd(int *fd, int fdtype)
 	switch (fdtype) {
 	case EFD_MEMFD:
 		return fcntl(*fd, F_ADD_SEALS, RUNC_MEMFD_SEALS);
-	case EFD_FILE: {
```

Comment:
It is removing the whitespace here as I have removed the `{ }` block (see description in the first commit).

Reply:
Sorry I was talking about a different hunk.

Reply:
I guess `indent` has a flag for this one, too, but I'm too lazy to look for one, since this change only happens once, so adding that flag won't result in drastic changes in the patch size, and I don't want to have too many flags.